scripts/tt-console/console.c

```diff
@@ -707,7 +709,7 @@ static int loop(struct console *const cons)
 		FD_ZERO(&fds);
 		FD_SET(STDIN_FILENO, &fds);
 
-		ret = select(STDIN_FILENO + 1, &fds, NULL, NULL, &tv);
```

Comment:
This should be the highest-numbered file descriptor in the set + 1. 

Of course, we all know that `STDIN_FILENO` is 0, but it's good to illustrate the relationship. I don't think there is any damage keeping it in.

Reply:
Fixed